test/test.py

```diff
@@ -46,20 +46,20 @@ def test_compile_error(self):
             process_tasks('compile_error/judge.yaml')
 
     def test_missing_input(self):
-        from task import InputMiss
-        with self.assertRaises(InputMiss) as raised:
+        from task import FileMiss
+        with self.assertRaises(FileMiss) as raised:
             process_tasks('missing_input/judge.yaml')
-        expect = re.compile("Input file for .*missing_input/testcases/output_1.txt is missing")
+        expect = "task.FileMiss: File in \"input\" type with index \"1\" is missing"
         message = f'{raised.exception}'
-        self.assertIsNotNone(expect.match(message), message)
+        self.assertIsNotNone(expect, message)
```

Comment:
TL;DR
If you are intended to compare these strings, use `assertEqual` instead and rename `message` as `exception`.

---

If you're trying to compare whether these strings are equal, you should use `assertEqual` instead.
In the previous version, `message` is the information for the assert fails but not the string that is going to compare.